src/clojure_tictactoe/game/game_play.clj

```diff
@@ -8,27 +8,28 @@
             [clojure-tictactoe.game.game-rules :as rules]
             ))
 
-(defn player-move [move-function board]
-  (board/mark-space (move-function board) @players/player-atom board))
+(defn player-move [move-function board marker]
+  (board/mark-space (move-function board) marker board))
+
+(defn continue-game [board markers turn]
+  (let [new-board (player-move (players/choose-player-function turn) board (markers (players/current-player turn)))]
```

Comment:
Newlines here would make this easier to follow.

Reply:
+1

Reply:
👍